backend/src/main/java/com/bakdata/conquery/models/events/Bucket.java

```diff
@@ -192,31 +175,41 @@ public CDateRange getAsDateRange(int event, Column column) {
 		};
 	}
 
+	public CDateRange getDateRange(int event, Column column) {
+		return ((DateRangeStore) getStore(column)).getDateRange(event);
+	}
+
 	public Object createScriptValue(int event, @NotNull Column column) {
 		return getStore(column).createScriptValue(event);
 	}
 
 	public Map<String, Object> calculateMap(int event) {
 		final Map<String, Object> out = new HashMap<>(stores.length);
+		Column[] columns = getTable().resolve().getColumns();
```

Comment:
resolve immernoch in hot-loop

Reply:
calculateMap wird pro Zeile aufgerufen

Reply:
Ich habe es jetzt in ein Lambda ausgelagert, dass pro bucket und cblock einmal generiert wird und stores und columns referenziert